test-requirements.txt

```diff
@@ -4,6 +4,7 @@ flake8-bugbear; python_version >= '3.5'
 flake8-pyi; python_version >= '3.6'
 lxml==4.2.4
 mypy_extensions>=0.4.0,<0.5.0
+typing_extensions>=3.7.0,<4.0.0
```

Comment:
I don't really understand why I needed to add this in as an explicit dependency.

All of the other pull requests seemed fine? But I wasn't able to import `typing_extensions` within the new `enums` plugin module without it...

Reply:
@Michael0x2a All the other files do:
```
MYPY=False
if MYPY:
    from typing_extensions import Final
```
I believe you don't have a guard for the import? I think doing the same guard would fix the issue.

Reply:
Argh, that would do it, thanks!

(I guess just blindly grepping for "typing_extensions" might not have quite given me the full picture...)